src/protocols/secure_channel/CASESession.cpp

```diff
@@ -645,6 +669,74 @@ CHIP_ERROR CASESession::RecoverInitiatorIpk()
     return CHIP_NO_ERROR;
 }
 
+#if INET_CONFIG_ENABLE_TCP_ENDPOINT
+void CASESession::HandleConnectionComplete(Transport::ActiveConnectionState * conn, CHIP_ERROR err)
+{
+    VerifyOrReturn(conn != nullptr);
+
+    CASESession * caseSession = reinterpret_cast<CASESession *>(conn->mAppState->appContext);
+    VerifyOrReturn(caseSession != nullptr);
+
+    // Exit and disconnect if connection setup encountered an error.
+    SuccessOrExit(err);
```

Comment:
move the parameter check to the beginning of the function.

Reply:
It is almost at the beginning. The CASESession object is just obtained before this so that it can be used in the exit label. I will add a log message there with the error.